File: 7/7.py
try:
    ahorros = float(input("Introduce la cantidad de ahorros en dolares: ($) "))
    if(type(ahorros) == float):
        for i in range(1,4):
            total = 0
            total = ahorros + (ahorros *0.3) * i   
            print("El total de ahorros tras el " + str(i) +".° año " + str(round(total,2)))   
           
except ValueError:
    print("Los valores deben ser numéricos")

File: 7/7.php
<?php
$ahorros = readline("Introduce la cantidad de ahorros en dolares: ($) ");
$locale = 'es_ES';
$nf = new NumberFormatter($locale,NumberFormatter::ORDINAL);
if(is_numeric($ahorros)){
    for($i=1;$i<=3;$i++){
        $total = $ahorros + ($ahorros * 0.3) * $i;
        $total = floatval($total);
        echo "El total de ahorros tras el ".$nf->format($i)." año ".round($total,2)."\n";
    }
    
}